Scripts/ParseEmailFiles/ParseEmailFiles.py

```diff
@@ -3505,6 +3505,7 @@ def handle_eml(file_path, b64=False, file_name=None, parse_only_headers=False, m
                 'Headers': header_list,
                 'HeadersMap': headers_map,
                 'Attachments': ','.join(attachment_names) if attachment_names else '',
+                'AttachmentNames': attachment_names if attachment_names else [],
```

Comment:
here you are adding new output, but in the context outputs you are replacing Attachments via AttachmentNames. It will break backward compatibility

Reply:
It will not break BC because i still populate the old path, just stop showing it in the outputs, there is no need for the changes you've sugested